xtask/src/config.rs

```diff
@@ -0,0 +1,63 @@
+use common::{make_path, path::PathsError, PathBuf};
+use normpath::BasePathBuf;
+use once_cell::sync::Lazy;
+
+pub static PATHS: Lazy<Paths> = Lazy::new(Paths::new);
```

Comment:
does this actually need a static? as far as I can see it could just be loaded at runtime and passed as a parameter to functions with no clear disadvantage

Reply:
I think I was mimicking the pattern from `CMD_STR` without thinking about the reason (being an externally called pre-defined hooked function.)

Good catch. I'll go look at that again, fix it, and look for other artifacts of that mental pattern!